emailTemplates/verification.html

```diff
@@ -0,0 +1,23 @@
+<html>
+  <head> </head>
+  <body>
+    <p>Hi there!</p>
+    <p>You are receiving this message to verify the email associated with your new SCCS account! Your username is: <strong>${username}</strong>.</p>
+    <p>
+      To access SCCS services, confirm your email by clicking: 
+      <!-- insert link with proper id and key -->
+      <a href="${domain}/account/reset?id=${resetId}&key=${resetKey}">this link</a>. Or, copy and
+      paste this link into your browser:
+    </p>
+    <p>${domain}/account/reset?id=${resetId}&key=${resetKey}</p>
+    <p>
+      (This link will stay active for seven days; after that, you'll have to
```

Comment:
This is actually something I forgot about—we'll need some logic to expire create-account requests properly, or maybe just set a TTL in mongodb. (And to not allow creation of an account that has the same username/email as one that's currently pending.)